ThingIFSDK/ThingIFSDKTests/GatewayAPILoginTests.swift

```diff
@@ -189,11 +187,6 @@ class GatewayAPILoginTests: GatewayAPITestBase {
             XCTAssertNil(error)
         }
 
-        XCTAssertThrowsError(try GatewayAPI.loadWithStoredInstance()) { error in
```

Comment:
@naoki-akimoto I have discussed with @satoshikumano , the API behavior is : 

if success then write to the `nsuserdefault`, if not then no writing.

So, there is no deletion if error happen.